Source/ExecutionEngine/CommandLineOptions.cs

```diff
@@ -1343,6 +1343,7 @@ protected override bool ParseOption(string name, CommandLineParseState ps)
               ps.CheckBooleanFlag("silent", x => Verbosity = CoreOptions.VerbosityLevel.Silent) ||
               ps.CheckBooleanFlag("traceTimes", x => TraceTimes = x) ||
               ps.CheckBooleanFlag("tracePOs", x => TraceProofObligations = x) ||
+              ps.CheckBooleanFlag("forceErrors", x => ForceBplErrors = x) ||
```

Comment:
Let us rename the flag to ```forceBplErrors``` to match the name of the internal field. Otherwise, I am happy to accept this PR.